tasks/base_tasks.py

```diff
@@ -1731,7 +1751,8 @@ def complete(self):
         if deps and not all([d.complete() for d in deps]):
             return False
 
-        return self._from_db(self.resource_id, self.version)[2] is not None
+        return self._from_db(self.resource_id, self.version)[2] is not None and \
```

Comment:
It appears only the filename is used from the output of `_from_db`. Maybe change it to return only that?

Reply:
Also, the previous deps code just checks if the `requires` have been met, i.e: `CreateRepoTable` has been run. We are usign custom `complete()` which we had a bad experience this just because of this?

I'm feeling a refactor: Create a new `Target` which is constructed with the needed parameters (repo table schema/name, resource_id and version), move this check there, and just use that target as an output. If the query fails (i.e: repo table does not exist), just rescue the error and return false.

All this assuming that our hatred for custom `complete()` implementations still holds :)

Reply:
Yes, we still hate custom `complete()` implementations. I have added a new `RepoTarget`. Thank you for the suggestion! :hugs: